tests/unit/records/marshmallow/test_marshmallow_base.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from datetime import datetime
+from unittest.mock import Mock
+
+import pytest
+from marshmallow import fields
+
+from inspirehep.records.marshmallow.base import (
+    BaseSchema,
+    ElasticSearchBaseSchema,
+    EnvelopeSchema,
+    wrapSchemaClassWithMetadata,
+)
+
+
+def test_base_schema():
+    class TestSchema(BaseSchema):
+        pass
+
+    data = {
+        "heroes": [{"name": "Jessica Jones"}, {"name": "Frank Castle"}],
+        "description": "A list of heroes",
+    }
+
+    result = TestSchema().dump(data).data
+
+    assert data == result
+
+
+def test_base_schema_with_exclude():
+    class TestSchema(BaseSchema):
+        class Meta:
+            exclude = ["description"]
+
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+            self.post_dumps.append(self.another_post_dump)
```

Comment:
isn't this testing only `exclude`?

Reply:
also this means we can't use `@post_dump` anymore for the subclasses? Wasn't aware of that :(